config/manifests/bases/openstack-operator.clusterserviceversion.yaml

```diff
@@ -48,4 +53,9 @@ spec:
   provider:
     name: Red Hat Inc.
     url: https://redhat.com/
+  relatedImages:
+  - image: registry.redhat.io/rhosp-rhel8/openstack-tripleoclient:16.2
```

Comment:
in OPENSTACKCLIENT_IMAGE_URL_DEFAULT and sample we use quay.io/tripleozedcentos9/openstack-tripleoclient:current-tripleo , does it make sense to set here a different image?

Reply:
The latest PR aligns these